# Definition of various tests
test_1:
  testblock_3s:
    time:
      - groundtruth: 3
        groundtruth_epsilon: 0.5
  testblock_5s:
    time:
      - groundtruth: 5
        groundtruth_epsilon: 0.5
  testblock_8s:
    time:
      - groundtruth: 8
        groundtruth_epsilon: 0.5
test_2:
  testblock_3s:
    time: []
  testblock_5s:
    time: []
  testblock_8s:
    time: []
